tests/blackbox/README.md

```diff
@@ -0,0 +1,22 @@
+# Blackbox Testing Framework
+
+Provides Evaluation tests for Kyma Companion and Validation of LLMs as judge for tests. Evaluation tests are used to measure the improvement of Kyma Companion, while Validation is used to find the best model to judge the test scenarios.
```

Comment:
```suggestion
The black box testing framework provides Evaluation tests for Kyma Companion and Validation of large language models (LLMs) as assessments for tests. Evaluation tests measure the improvement of Kyma Companion, while Validation finds the best model to verify the test scenarios.
```

Reply:
If `Evaluation` and `Validation` are not proper names, please change all the mid-sentence occurrences to lowercase.